core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -409,6 +409,52 @@ class Router(formatOps: FormatOps) {
                 .withIndent(2, expire, Left)
             )
         }
+
+      // Parameter opening for one parameter group. This format works
+      // on a group-by-group basis.
```

Comment:
Unfortunately no. The design is intentionally low level, for better or worse. Policies are the way to achieve what you want, which is to enforce behavior in upcoming format tokens. The first open paren of a def is where you enable the policy for the entire def/class parameter list.